docker/package/model.py

```diff
@@ -681,7 +681,6 @@ def __init__(
 
     def fetch_sources(self, out_dir, binaries_dir=None):
         os.makedirs(out_dir)
-        package_dir = out_dir
         subprocess.call(
             f"cp -r {os.path.dirname(__file__)}/baking/* {package_dir}/", shell=True
```

Comment:
```suggestion
            f"cp -r {os.path.dirname(__file__)}/baking/* {out_dir}/", shell=True
```

Reply:
BTW, aside from this comment the rest LGTM, so feel free to squash the changes as well @krendelhoff2